docs/ci.md

```diff
@@ -84,13 +84,13 @@ The project currently supports the usage of the following runners for the E2E jo
 | Area | Feature | [`e2e.yml`](https://github.com/instructlab/instructlab/blob/main/.github/workflows/e2e.yml) (small) ![`e2e.yaml` on `main`](https://github.com/instructlab/instructlab/actions/workflows/e2e.yml/badge.svg?branch=main)| [`e2e-nvidia-t4-x1.yml`](https://github.com/instructlab/instructlab/blob/main/.github/workflows/e2e-nvidia-t4-x1.yml) (small) | [`e2e-nvidia-a10g-x1.yml`](https://github.com/instructlab/instructlab/blob/main/.github/workflows/e2e-nvidia-a10g-x1.yml) (medium) | [`e2e-nvidia-a10g-x4.yml`](https://github.com/instructlab/instructlab/blob/main/.github/workflows/e2e-nvidia-a10g-x4.yml) (large) ![`e2e-nvidia-a10g-x4.yaml` on `main`](https://github.com/instructlab/instructlab/actions/workflows/e2e-nvidia-a10g-x4.yml/badge.svg?branch=main)|
 | --- | --- | --- | --- | --- | --- |
 | **Serving**  | llama-cpp                |✅|✅|✅|✅ (temporary)|
-|              | vllm                     |⎯|⎯|⎯|❌|
+|              | vllm                     |⎯|⎯|⎯|✅|
 | **Generate** | simple                   |✅|✅|✅|⎯|
 |              | full                     |⎯|⎯|⎯|✅|
-| **Training** | legacy+Linux             |⎯|⎯|✅|⎯|
+| **Training** | legacy+Linux             |⎯|⎯|❌|⎯|
```

Comment:
This [run](https://github.com/instructlab/instructlab/actions/runs/10085764596/job/27887223060) reflects the reason for this change.

CC @RobotSail

Reply:
@alimaredia You need to update the `--data-path` to point to the correct file path - since we run each end-to-end in a temporary directory, you have to use this directory when running the training.

Reply:
Thanks! That can get changed in another PR focused on getting the A10x1 workflow working, I just wanted to highlight the reason for the change